src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "symfony/phpunit-bridge",
     "type": "symfony-bridge",
     "description": "Provides utilities for PHPUnit, especially user deprecation notices management",
-    "keywords": [],
+    "keywords": ['dev'],
```

Comment:
Are single quotes valid json?

Reply:
No, but @fabpot fixed it while merging in e86d26abd07187d5ec5b2d780cfd1d872d57fe2a

Reply:
Not yet :)
I fixed it just after the merge when I realized that ;)